docs/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Components with a big letter like MenuItem and ValidationMessage are broken.
You make /menuItem in the nav link but the original is /menu-item

Reply:
fixed ... freaking gatsby